testcases/network/rpc/rpc-tirpc/tests_pack/rpc_svc_1/rpc_svc_1.c

```diff
@@ -103,10 +103,10 @@ char *simplePing(union u_argument *inVar, SVCXPRT * transp)
 char *svc_getcaller_test(union u_argument *inVar, SVCXPRT * transp)
 {
 	//In this function we test svc_getcaller function basically (simple call)
-	struct sockaddr_in *sa = NULL;
+	struct sockaddr *sa = NULL;
 	static int result;
 
-	sa = svc_getcaller(transp);
+	sa = (struct sockaddr *) svc_getcaller(transp);
```

Comment:
The casting would be needed in many RPC tests:
```
$ git grep -l sockaddr_in testcases/network/rpc/ | wc -l
39
```
It would make sense to fix them all (with sed or awk scripting to make editing fast).
FYI TI-RPC tests are in terrible shape (code duplicity, lots of compiler warnings, useless comments), but upstream does not care.